supporting_content/columns/skupriceid.md

```diff
@@ -40,13 +40,13 @@ The following table serves as the basis for reviewing the SkuPriceId spec, as we
 
 | ChargeCategory | ChargeClass | perSku/bulk                       | SkuId            | SkuPriceId       |
 |----------------|-------------|-----------------------------------|------------------|------------------|
-| Usage          | Regular     | MUST be perSku and perSkuPrice    | MUST not be null | MUST not be null |
+| Usage          | (null)      | MUST be perSku and perSkuPrice    | MUST not be null | MUST not be null |
```

Comment:
nit: Can we leave this in supporting content to convey the values we've discussed? Alternatively, we could just keep this as a separate note and call out options for future consideration.
```suggestion
| Usage          | Regular/Standard/Original/Direct/(null) | MUST be perSku and perSkuPrice    | MUST not be null | MUST not be null |
```